backend/modules/metadata_store/prisma_store.py

```diff
@@ -64,32 +70,26 @@ async def aget_collection_by_name(
         self, collection_name: str, no_cache: bool = True
     ) -> Optional[Collection]:
         try:
-            collection: Optional[
-                "PrismaCollection"
-            ] = await self.db.collection.find_first(where={"name": collection_name})
-            if collection:
```

Comment:
No need to check if the collection exists since Prisma will raise a `RecordNotFound` exception if the record does not exist with the given name.